sql/src/test/resources/drill/window/queries/druid_queries/partition_by_array/wikipedia_query_2.q

```diff
@@ -0,0 +1,6 @@
+select
+countryName, cityName, channel,
+row_number() over (partition by array[1,2,3] order by countryName) as c
+from wikipedia
+where countryName in ('Austria', 'Republic of Korea')
+group by countryName, cityName, channel
```

Comment:
dont think this test adds much of a value

Reply:
@sreemanamala I didn't see such a query tested in existing tests, I think it's good to have a basic test for such queries as well. Doesn't hurt 😅